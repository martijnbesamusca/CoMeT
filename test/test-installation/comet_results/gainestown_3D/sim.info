{'cmdline': ['/home/martijn/Documents/uva/thesis/CoMeT/run-sniper',
             '-v',
             '-s',
             'memTherm_core',
             '-n',
             '4',
             '-c',
             'gainestown_3D',
             '--',
             '/home/martijn/Documents/uva/thesis/CoMeT/test/test-installation/test'],
 'git_revision': '70a94ea0a46a09195fc7cc97e7411ec90733f6a3',
 'host': '114c1c70b953',
 'pin_version': 'unknown',
 'rusage': (21.45714,
            1.323215,
            72364,
            0,
            0,
            0,
            38232,
            0,
            0,
            0,
            2792,
            0,
            0,
            0,
            247546,
            268),
 'snipercmd': ['bash',
               '-c',
               '/home/martijn/Documents/uva/thesis/CoMeT/lib/sniper -c /home/martijn/Documents/uva/thesis/CoMeT/config/base.cfg --general/total_cores=4 --general/output_dir=/home/martijn/Documents/uva/thesis/CoMeT/test/test-installation --config=/home/martijn/Documents/uva/thesis/CoMeT/config/nehalem.cfg --config=/home/martijn/Documents/uva/thesis/CoMeT/config/gainestown_3D.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/home/martijn/Documents/uva/thesis/CoMeT/test/test-installation/sim.scripts.py -g --hooks/script0args= -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmpxrm2aT/run_benchmarks'],
 't_elapsed': 26.03582501411438,
 't_start': 1653910861.000604,
 'tracecmds': [['/home/martijn/Documents/uva/thesis/CoMeT/record-trace',
                '-o',
                '/tmp/tmpxrm2aT/run_benchmarks',
                '-v',
                '--pa',
                '-e',
                '1',
                '-s',
                '0',
                '-r',
                '1',
                '--follow',
                '--routine-tracing',
                '--',
                '/home/martijn/Documents/uva/thesis/CoMeT/test/test-installation/test']],
 'user': None,
 'vmem': 453079040L}
