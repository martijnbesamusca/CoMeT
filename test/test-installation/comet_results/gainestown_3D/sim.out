                                     | Core 0     | Core 1     | Core 2     | Core 3    
  Instructions                       |   21246362 |          0 |          0 |          0
  Cycles                             |    8783280 |    8783280 |    8783280 |    8783280
  IPC                                |       2.42 |       0.00 |       0.00 |       0.00
  Time (ns)                          |    2439800 |    2439800 |    2439800 |    2439800
  Idle time (ns)                     |        -69 |    2439800 |    2439800 |    2439800
  Idle time (%)                      |      -0.0% |     100.0% |     100.0% |     100.0%
Branch predictor stats               |            |            |            |           
  num correct                        |    2597554 |          0 |          0 |          0
  num incorrect                      |      26947 |          0 |          0 |          0
  misprediction rate                 |      1.03% |      0.00% |      0.00% |      0.00%
  mpki                               |       1.27 |       0.00 |       0.00 |       0.00
TLB Summary                          |            |            |            |           
  I-TLB                              |            |            |            |           
    num accesses                     |    2890054 |          0 |          0 |          0
    num misses                       |         46 |          0 |          0 |          0
    miss rate                        |      0.00% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.00 |       0.00 |       0.00 |       0.00
  D-TLB                              |            |            |            |           
    num accesses                     |    7876038 |          0 |          0 |          0
    num misses                       |        907 |          0 |          0 |          0
    miss rate                        |      0.01% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.04 |       0.00 |       0.00 |       0.00
  L2 TLB                             |            |            |            |           
    num accesses                     |        953 |          0 |          0 |          0
    num misses                       |        437 |          0 |          0 |          0
    miss rate                        |     45.86% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.02 |       0.00 |       0.00 |       0.00
Cache Summary                        |            |            |            |           
  Cache L1-I                         |            |            |            |           
    num cache accesses               |    2890054 |          0 |          0 |          0
    num cache misses                 |        920 |          0 |          0 |          0
    miss rate                        |      0.03% |       inf% |       inf% |       inf%
    mpki                             |       0.04 |        inf |        inf |        inf
  Cache L1-D                         |            |            |            |           
    num cache accesses               |    7876038 |          0 |          0 |          0
    num cache misses                 |      49513 |          0 |          0 |          0
    miss rate                        |      0.63% |       inf% |       inf% |       inf%
    mpki                             |       2.33 |        inf |        inf |        inf
  Cache L2                           |            |            |            |           
    num cache accesses               |      50917 |          0 |          0 |          0
    num cache misses                 |      48991 |          0 |          0 |          0
    miss rate                        |     96.22% |       inf% |       inf% |       inf%
    mpki                             |       2.31 |        inf |        inf |        inf
DRAM summary                         |            |            |            |           
  num dram accesses                  |      20364 |      20781 |      20314 |      20317
  average dram access latency (ns)   |      23.53 |      29.19 |      23.50 |      23.60
  average dram queueing delay        |       0.10 |       5.76 |       0.07 |       0.17
  average dram bandwidth utilization |      7.00% |      7.00% |      7.00% |      7.00%
Coherency Traffic                    |            |            |            |           
  num loads from dram                |      16363 |          0 |          0 |          0
  num loads from dram cache          |          0 |          0 |          0 |          0
  num loads from remote cache        |          0 |          0 |          0 |          0
